homeassistant/components/fritzbox/sensor.py

```diff
@@ -62,6 +63,19 @@ async def async_setup_entry(
                     ain,
                 )
             )
+            entities.append(
```

Comment:
```suggestion

        if device.rel_humidity is not None:
            entities.append(
```

Reply:
`FritzBoxHumiditySensor` should only be added, when device provides a `rel_humidity` value

Reply:
That was exactly my concern. I added the `None`-check.